eosdis-nasa/earthdata-pub-forms
# Edpub 1223 add upload module

# Description

Addes the upload module to the dependencies and build plan

## Spec

See Ticket: [EDPUB-1223](https://bugs.earthdata.nasa.gov/browse/EDPUB-1223)

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Run ```npm run i```
4. Confirm the ```edpub-data-upload-utility``` is present tin the ```node_modules``` folder
5. Run this branch in bamboo and confirm it passes
---

## Change Log

Addes the upload module to the dependencies and build plan

* [Add _____](link to commitid)
* [Fix _____](link to commitid)
